lang/main.json

```diff
@@ -422,5 +422,15 @@
         "seconds": "__count__s",
         "speakerStats": "Speaker Stats",
         "speakerTime": "Speaker Time"
+    },
+    "deviceSelection": {
+        "audioInput": "Audio input",
+        "audioOutput": "Audio output",
+        "deviceSelection": "Video settings",
+        "noOtherDevices": "No other devices available",
+        "noThanks": "No thanks",
+        "selectADevice": "Select a device",
+        "testAudio": "Test sound",
+        "videoInput": "Video input"
```

Comment:
We need to remove the previously used strings. I believe they are under  "settings":
"selectCamera": "Camera",
"selectMic": "Microphone",
"selectAudioOutput": "Audio output",
and anything else that was used in the previous UI and is no more used here.